src/mount/readdata_cache.h

```diff
@@ -24,22 +24,45 @@
 
 #include "common/small_vector.h"
 #include "common/time_utils.h"
+#include "mount/memory_info.h"
```

Comment:
Please, check if this header is really needed.

Reply:
Done